discovery.go

```diff
@@ -96,7 +88,7 @@ func (d *discovery) advertise(namespaces []string) time.Duration {
 	for _, provides := range namespaces {
 		_, err = d.rd.Advertise(d.ctx, provides)
 		if err != nil {
-			log.Debugf("failed to advertise %q in the DHT: %s", provides, err)
```

Comment:
This should probably be a warning level message, but we see it frequently in tests (kbucket.ErrLookupFailure - failed to find any peer in table) when a bootnode or swapd instance has an empty bootnode list. Here in discovery, we don't have access to the bootnode list to handle different levels based on the error message.